nilearn/version.py

```diff
@@ -1 +1,97 @@
+# *- encoding: utf-8 -*-
+"""
+nilearn version, required package versions, and utilities for checking
+"""
+# Author: Alexandre Abraham, Philippe Gervais, Ben Cipollini
+# License: simplified BSD
+
 __version__ = '0.1b1'
+
+# All metadata needed for required modules:
+#    * minver: minimum required version of the module
+#    * manual: whether the module must be installed manually
+# See more info in check_module_dependencies docstring.
+#
+# This is a list to preserve order, and to avoid the Python 2.7
```

Comment:
It's not a list but a tuple (to be honest a list would be fine too, mutable vs immutable is not so crucial here). 

I would also make it more explicit that we care about order because we want to check required dependencies in a meaningful order (from most core/common packages to less core/common ones roughly). In this regards gzip should probably be first in the list.